content/en/tests/browser_tests.md

```diff
@@ -37,7 +37,7 @@ RUM integration is supported for Cypress browser tests and Selenium-driven brows
 
 #### Selenium
 
-* `selenium-js` >= 4.11.0, `dd-trace-js` >= 5.11.0
+* `selenium-js` >= 4.11.0, `dd-trace-js` >= 5.11.0 / >= 4.35.0 / >= 3.56.0
```

Comment:
thanks! let's remove v3 since it's no longer maintained (I'll need to change some other places too)
```suggestion
* `selenium-js` >= 4.11.0, `dd-trace-js` >= 5.11.0 / >= 4.35.0
```